crates/wdk-build/src/lib.rs

```diff
@@ -19,7 +19,7 @@ pub mod cargo_make;
 pub mod metadata;
 /// Module for utility code related to the cargo-make experience for building
```

Comment:
If you're keeping this module doc comment, update it to reflact that its contents is meant to be internal and private (ie. not exported from crate)

Reply:
Moved the doc comment to utils module and removed it from lib.rs